custom/icds/tasks.py

```diff
@@ -110,26 +110,14 @@ def get_language_code(user_id, telugu_user_ids, marathi_user_ids):
         return HINDI
 
 
-@periodic_task(
-    run_every=crontab(hour=9, minute=0),
-    queue=settings.CELERY_PERIODIC_QUEUE,
-    ignore_result=True
-)
-def run_user_indicators(phased_rollout=True):
+def run_user_indicators(phased_rollout=True, run_weekly=False, run_monthly=False):
     """
     Runs the weekly / monthly user SMS indicators at 9am IST.
     This task is run every day and the following logic is applied:
         - if it's Monday, the weekly indicators are sent
         - if it's the first of the month, the monthly indicators are sent
         - if it's neither, nothing happens
```

Comment:
is this docstring still correct? it looks like the decision of which days to run the subtasks is now an external decision, and whether they will be run or not depends on the values of the arguments `run_weekly` and `run_monthly`

Reply:
@millerdev it's not correct, and i'm actually going to be removing this code in a week. i'm only leaving this function around in case i need to manually do things the old way in the next few days. in the meantime i've removed the docstring to avoid any confusion